viz_scripts/scaffolding.py

```diff
@@ -105,7 +105,7 @@ def data_quality_check(expanded_ct):
 
 def unit_conversions(df):
     df['distance_miles']= df["distance"]*0.00062 #meters to miles
-    df['duration'] = df['end_ts'] - df['start_ts']
+    df['duration_h'] = (df['end_ts'] - df['start_ts']) / 60 / 60 #seconds to hours
```

Comment:
@styoung3 why do you need to have the duration in hours? We are using speeds from the dataset directly - it's not like we need to convert use an external mph value.

Reply:
I calculate average speed using (distance / duration) for each trip. Then I aggregate by mode and calculate the mean. MPH was easier for me to understand.